browser/src/RegionalGenomicConstraintTrack.tsx

```diff
@@ -0,0 +1,296 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import queryString from 'query-string'
+
+// @ts-expect-error TS(7016) FIXME: Could not find a declaration file for module '@gno... Remove this comment to see the full error message
+import { Track } from '@gnomad/region-viewer'
+import { TooltipAnchor } from '@gnomad/ui'
+import Link from './Link'
+
+import InfoButton from './help/InfoButton'
+
+const Wrapper = styled.div`
+  display: flex;
+  margin-bottom: 1em;
+`
+
+const PlotWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 100%;
+`
+
+const RegionAttributeList = styled.dl`
+  margin: 0;
+
+  div {
+    margin-bottom: 0.25em;
+  }
+
+  dt {
+    display: inline;
+    font-weight: bold;
+  }
+
+  dd {
+    display: inline;
+    margin-left: 0.5em;
+  }
+`
+
+function regionColor(region: any) {
+  // http://colorbrewer2.org/#type=sequential&scheme=YlOrRd&n=3
+  // https://colorbrewer2.org/#type=sequential&scheme=PuBu&n=4
+  let color
+  if (region.z > 2.5) {
+    color = '#f03b20'
+  } else if (region.z > 1.2) {
+    color = '#feb24c'
+  } else if (region.z > 0.6) {
+    color = '#ffeda0'
+  } else if (region.z > -0.6) {
+    color = '#e2e2e2'
+  } else if (region.z > -1.2) {
+    color = '#bdc9e1'
+  } else if (region.z > -2.5) {
+    color = '#74a9cf'
+  } else {
+    color = '#0570b0'
+  }
+  return color
+}
+
+const LegendWrapper = styled.div`
+  display: flex;
+
+  @media (max-width: 600px) {
+    flex-direction: column;
+    align-items: center;
+  }
+`
+
+const Legend = () => {
+  const currentParams = queryString.parse(location.search)
```

Comment:
Let's do `const variantId: string? = queryString.parse(location.search)`. We then don't have to do all the `currentParams.variant as string` business below. Also, the codebase currently uses e.g. `variant` and `variantId` interchangably, which is a common antipattern. The name of a thing is not the thing. As such, we should use each respectively where appropriate, and here we're concerned with the ID of a variant, not a variant. Hence the identifier changes to `variantId`.